qemu/tests/cfg/boot_from_virtiofs.cfg

```diff
@@ -18,6 +18,7 @@
         image_name_stg = images/stg
         remove_image_stg = yes
         force_create_image_stg = yes
+        bootindex_stg = 2
```

Comment:
It works. But I'm curious how about setting boot index for virtiofs device, this might avoid the other storages setting bootindex if needed.

Reply:
In this case, viriofs device is already set as ` fs_driver_props = {"bootindex": 1}`.

Reply:
Sorry, I didn't notice it before.
Then it looks good to me.

Reply:
Hi @hellohellenmao 
I checked the history of this file https://github.com/autotest/tp-qemu/pull/3857/files .
But there are no errors with the previous version from the conversation,no such issue: `[qemu output] qemu-kvm: -device {"id": "vufs_virtiofs_fs", "chardev": "char_virtiofs_fs", "tag": "myfs", "bootindex": 1, "driver": "vhost-user-fs-pci", "bus": "pcie-root-port-3", "addr": "0x0"}: The bootindex 1 has already been used`
Could you help me to check if it worked with the previous version? @leidwang CC @xiagao 
I just wonder why it passed before and whether there are product changes.
Thanks.

Reply:
Good question. From my testing, the current code works fine with virtio-scsi, but not virtio-blk device. And I have checked https://github.com/autotest/tp-qemu/pull/3857/files, the testing device is also virtio-scsi in the comment.

Reply:
> Hi @hellohellenmao I checked the history of this file https://github.com/autotest/tp-qemu/pull/3857/files . But there are no errors with the previous version from the conversation,no such issue: `[qemu output] qemu-kvm: -device {"id": "vufs_virtiofs_fs", "chardev": "char_virtiofs_fs", "tag": "myfs", "bootindex": 1, "driver": "vhost-user-fs-pci", "bus": "pcie-root-port-3", "addr": "0x0"}: The bootindex 1 has already been used` Could you help me to check if it worked with the previous version? @leidwang CC @xiagao I just wonder why it passed before and whether there are product changes. Thanks.

Hi @YongxueHong 

I agree with you, maybe we can check our previous testing results.To check when we hit this issue first, and if this issue is a product regression issue.

Thanks!